### DISCUSSION TITLE
Can this be used to self-host netbird?

### DISCUSSION THREAD

**Discussion by CumpsD** [dominion-solutions/ansible-netbird#31] [Q&A]
How would I use this Ansible role to self-host netbird?

**Comment by spam-n-eggs** [comment]
@CumpsD I'm glad to see someone more than me is using this library!  We'll spec out what we can do.  Expect to see an update in Q2 2025.

**Reply by CumpsD** [reply]
Thanks! I just found out about netbird, looks great :)
